src/app/features/item/item-template/item-preview.service.ts

```diff
@@ -0,0 +1,1394 @@
+import { Injectable } from '@angular/core';
+import { MysqlQueryService } from '@keira-shared/services/mysql-query.service';
+import { ItemTemplateService } from './item-template.service';
+import { SqliteQueryService } from '@keira-shared/services/sqlite-query.service';
+import { ITEM_TYPE, ITEM_MOD } from '@keira-shared/constants/options/item-class';
+import { ITEM_CONSTANTS } from './item-constants';
+import { MAX_LEVEL, lvlIndepRating, gtCombatRatings, CLASSES, RACE, resistanceFields } from './item-preview';
+import { ITEM_FLAG } from '@keira-shared/constants/flags/item-flags';
+import { ITEMS_QUALITY } from '@keira-shared/constants/options/item-quality';
+
+@Injectable()
+export class ItemPreviewService {
+  private readonly ITEM_CONSTANTS = ITEM_CONSTANTS;
+
+  /* istanbul ignore next */ // because of: https://github.com/gotwarlost/istanbul/issues/690
+  constructor(
+    private readonly editorService: ItemTemplateService,
```

Comment:
remove the `ItemTemplateService` dependency. All you need is an instance of `ItemTemplate`. No need to involve forms either.